modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -3,77 +3,196 @@
 
-The Couchbase Node.js SDK 3.0 is a complete rewrite of the API, reducing the number of overloads to present a simplified surface area, and adding support for future Couchbase Server features like Collections and Scopes (available in Couchbase Server 6.5 as a xref:concept-docs:collections.adoc[developer preview]).
-The 3.0 SDK also brings in `promises`, to reduce the complexity of asynchronous javascript in client applications, as well as extending the management APIs and bringing better debugging and logging options
+The Couchbase Node.js SDK 3.0 is a complete rewrite of the API, reducing the number of overloads to present a simplified surface area, and adding support for future Couchbase Server features like Collections and Scopes (available in Couchbase Server 6.5 as xref:concept-docs:collections.adoc[a developer preview]).
 
+The 3.0 SDK also brings in link:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises[promises], to reduce the complexity of asynchronous javascript in client applications, as well as extending the management APIs and bringing better debugging and logging options
 
 // tag::prep[]
 
 // end::prep[]
 
 // tag::install[]
 
+== Creating a New Node.js Project
+// This section is optional
+
+Creating a Node.js project is as easy as making a directory and initializing it with npm. The next two commands will do that for us. Open up a terminal and run the following command:
+
+[source,bash]
+----
+mkdir node-couchbase-project && cd $_
+----
+
+The command above will make our directory and change our current working directory.
+
+[source,bash]
+----
+npm init -y
+----
+
+If a directory does not already have a `package.json` at its root, this means it is not initialized. The command above will accomplish this. 
+
+Note: We have used the `-y` flag to take the initialization defaults. To change any of these defaults, just open the `package.json` and manually make any changes. 
+
+[source,json]
+----
+{
+  "name": "node-couchbase-project",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC"
+}
+----
+
 == Installing the SDK
 
-[source,javascript]
+[source,bash]
 ----
-npm install couchbase
+npm install couchbase --save
 ----
 
+Note: This will download the latest Couchbase Node.js SDK, and add a dependency to your `package.json`.
+
 // end::install[]
 
-Information on new features, fixes, known issues, as well as information on how to install older release versions is xref:project-docs:sdk-release-notes.adoc[in the release notes].
+Information on new features, fixes, known issues, as well as information on how to install older release versions is in the link:https://docs.couchbase.com/nodejs-sdk/3.0/project-docs/sdk-release-notes.html[release notes].
```

Comment:
Recommend using xref instead of link as it will resolve to the current version in the future (like v3.1) as well.

Reply:
I am not sure how that got changed. I don't remember changing the xref to link, very weird. I will change it back to: xref:project-docs:sdk-release-notes.adoc[in the release notes].

Reply:
Changes have been made to fix this: https://github.com/couchbase/docs-sdk-nodejs/pull/74/commits/4b05605e08fef694ee3d05d6aa946d07b74d9964